docker-compose-dev.yml

```diff
@@ -1,19 +1,12 @@
 services:
-  caddy:
```

Comment:
Does this just work out of the box? I don't see an nginx config included with this PR.